contracts/src/Teleporter/upgrades/README.md

```diff
@@ -45,23 +41,30 @@ contract ERC20Bridge is
             .getBlockchainID();
     }
     ...
+    function _receiveTeleporterMessage(
```

Comment:
Perhaps add some comments to the example application? I expect many readers will jump straight to the example code, so adding comments here would be helpful.